src/features/unified-share-modal/EmailForm.js

```diff
@@ -205,14 +207,27 @@ class EmailForm extends React.Component<Props, State> {
         const { intl } = this.props;
         let contactsFieldError = '';
 
-        if (text && !this.isValidEmail(text)) {
+        if (text && !emailValidator(text)) {
```

Comment:
`isValidContactPill` has a slightly different purpose than `isValidEmail`, so I think it no longer makes sense to call it here.